### ISSUE TITLE
Sustain and release issue

### ISSUE THREAD

**Issue by acoto87** [schellingb/TinySoundFont#43]
If I call `tsf_channel_note_on(soundFont, channel, key, 0);` internally the library call `tsf_note_off(f, preset_index, key);` which causes the notes for the channel to go off, but applying sustain (some kind of fading out I think).

I'm using that first call as a way to mute the music of my game, is there any way the sustain and release doesn't get applied? Or should I approach the muting of the music in another way?

I also have this problem when I change the music, say Music 1 is playing and the I as a player change it to Music 2, you can hear the fading out of Music 1 while the Music 2 starts to play.

Am I doing something wrong, using the library wrong or something?

**Comment by tobybear** [comment]
Maybe these make the difference:
TSFDEF void tsf_channel_note_off_all(tsf* f, int channel); //end with sustain and release
TSFDEF void tsf_channel_sounds_off_all(tsf* f, int channel); //end immediatly

**Comment by acoto87** [comment]
Yeah, `tsf_channel_sounds_off_all` interally call `tsf_voice_endquick` which is what I would want in this case when I call `tsf_channel_note_on(soundFont, channel, key, 0);`. Maybe I'm not suppose to use `0` volume as a mean to mute a sound and instead I need to call some variant that of `tsf_channel_note_off_all` or `tsf_channel_sounds_off_all`.

Thanks for the response :)
